charts/logzio-telemetry/values.yaml

```diff
@@ -288,6 +288,9 @@ metricsConfig:
         p8s_logzio_name: ${P8S_LOGZIO_NAME}
       headers:
         Authorization: "Bearer ${METRICS_TOKEN}"
+      remote_write_queue:
```

Comment:
Why like this and not as parameters ?

Reply:
These values can be changed directly with --set metricsConfig.exporters.prometheus...
The other options is to set it as environment variable or render this section in the pod template file.